src/ConnectionGraphicsObject.cpp

```diff
@@ -205,7 +205,7 @@ mouseReleaseEvent(QGraphicsSceneMouseEvent* event)
   event->accept();
 
```

Comment:
We can get then view by using `auto view = static_cast<QGraphicsView *>(event->widget());`

Reply:
Sorry i didn't get it